packages/block-editor/src/components/collab/toolbar.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { collabComment } from '@wordpress/icons';
+const openCollabBoard = () => {};
+
+export default function BlockCommentToolbar( {} ) {
+	return (
+		<>
+			<ToolbarGroup className="block-editor-block-comment-toolbar">
+				<ToolbarButton
+					accessibleWhenDisabled
+					onClick={ openCollabBoard }
+					icon={ collabComment }
+					label={ __( 'Comment' ) }
+				/>
+			</ToolbarGroup>
+		</>
```

Comment:
No need for this `React.Fragment` if there's only one root React element

```suggestion
		<ToolbarGroup className="block-editor-block-comment-toolbar">
			<ToolbarButton
				accessibleWhenDisabled
				onClick={ openCollabBoard }
				icon={ collabComment }
				label={ __( 'Comment' ) }
			/>
		</ToolbarGroup>
```